src/main/java/optic_fusion1/mcantimalware/Main.java

```diff
@@ -52,8 +53,9 @@
     private FileConfiguration checksumDatabase;
     private boolean scanSingleFile;
     private OptionSet options;
-
-    private void init() {
+    public static String version = "3.57";
```

Comment:
This field should be final and probably not static.

Reply:
That was only  temp for now, I talked with OpticFusion on discord and as  soon as the logger is done I will get the version via a maven property

Reply:
Logger is pretty much done besides issues #152 (due to things related to the new logger) and #153

Reply:
Logger is done, you'll want to work on the conflicts though :P